opencog/guile/SchemeSmob.cc

```diff
@@ -224,14 +224,7 @@ void SchemeSmob::module_init(void*)
 {
 	// The portion of (opencog) done in C++
 	register_procs();
-	// XXX FIXME This is a hack for testing.
-	// It avoids requiring a manual change of guile load path.
-	// Alternatives to always adding this path would be:
-	// - make ctest manage the environment variable GUILE_LOAD_PATH during testing
-	// - set GUILE_LOAD_PATH env var in each test, before initialising Scheme
-	// - pass an optional load path through SchemeEval/SchemeSmob
-	//scm_c_eval_string("(add-to-load-path \"" PROJECT_BINARY_DIR "/opencog/scm\")");
-
+	
```

Comment:
Apparently the tests only passed locally due to having an installed version of atomspace.

Meanwhile, leaving the hack in place would make my opencog build point at the atomspace build dir when testing (instead of the system atomspace).

Thus I will do the work to make the AtomSpace tests pass without this.

Reply:
Yay!